third_party/silabs/silabs_board.gni

```diff
@@ -69,6 +69,13 @@ declare_args() {
   chip_enable_ble_rs911x = use_rs9116 || use_SiWx917
 }
 
+// TODO - This needs to be removed once multiplexing issues resolved
```

Comment:
This isn't how you add a comment in a gni.

Reply:
Added correctly.